_episodes/13-scheduler.md

```diff
@@ -126,6 +137,103 @@ hostname
 ```
 {: .output}
 
+Let's try running it:
+
+```
+$ example-job.sh 
+```
+{: .language-bash}
+
+```
+bash: example-job.sh: command not found...
+```
+{: .error}
+
+Strangely enough, Bash can't find our script. As it turns out, Bash will only
+look in certain directories for scripts to run. To run anything else, we need
+to tell Bash exactly where to look. To run a script that we wrote ourselves, we
+need to tell Bash where it is explicitly, so it doesn't go searching. We could
+do this one of two ways: either with the absolute path 
+`{{ site.workshop_host_homedir }}/yourUserName/example-job.sh` (equivalently,
+`~/example-job.sh`), or with the relative path `./example-job.sh`.
+
+```
+$ ./example-job.sh
+```
+{: .language-bash}
+
+```
+bash: ./example-job.sh: Permission denied
+```
+{: .error}
+
+There's one last thing we need to do. Before a file can be run, it needs
+"permission" to execute. Let's look at our file's permissions with `ls -l`:
+
+```
+$ ls -l *.sh
+```
+{: .language-bash}
+
+```
+...
+-rw-rw-r-- 1 yourUsername yourGroup  40 Jan 16 19:41 example-job.sh
+...
+```
+{: .output}
+
+That's a huge amount of output: a full listing of everything in the directory.
```

Comment:
The command above only listed files with the `.sh` extension, and this sentence/paragraph should to be modified to reflect this.

Reply:
:+1: